### PR TITLE
Refine spin

### PR DESCRIPTION
**Pull request by EmceeEscher**
Toned down the spin after a collision by a lot. I actually prefer the original version, where they spin a lot, but Alla specifically requested us to make it less dramatic...

### CODE REVIEW

**Comment by theochit** [comment]
I think this definitely satisfies what Alla wants, but I miss the crazy spin too. I think the dramatic spin is okay, as long as we use a function of time to make the spin decrease. The way I took it, she was commenting on the fact that it would go from spinning fast to completely still with no intermediate stage

**Review by theochit — APPROVED** [review]

**Comment by PenguinFlyingHigh** [comment]
Spinouts slow down with time.

File: src/car.cpp [hunk 1]
```diff
@@ -300,7 +301,8 @@ void Car::update(float ms)
 			if (sqrt(m_velocity.x * m_velocity.x + m_velocity.y * m_velocity.y > 10.f)) {
```
Comment (theochit) [inline src/car.cpp]: Doesn't seem to make a difference, but I think the parentheses are off here.

Comment (EmceeEscher) [reply src/car.cpp]: oh good catch. also wtf, how did this work at all...

**Pull request merged**